05/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles/reset.css">
+    <link rel="stylesheet" href="./styles/mobile.css">
+    <link rel="stylesheet" media="(min-width: 800px)" href="./styles/desktop.css">
+</head>
+<body>
+    <div class="container">
+        <header>
+            <h1><img src="./images/logos.png" alt="logo"></h1>
+            <label for="menu-switcher">|||</label>
+            <input type="checkbox" id="menu-switcher">  
+            <ul class="menu">
+                <li class="menu__item"><a class="menu__link" href="#">O mnie</li></a>
+                <li class="menu__item"><a class="menu__link" href="#">Portfolio</li></a>
+                <li class="menu__item"><a class="menu__link" href="#">Kontakt</li></a>
+            </ul>
+        </header>
+    </div>
+    <div class="container">
```

Comment:
Raczej zamiast `<div>` wstawiłbym `<section>`. a wew. miałbym np. `<main>`.
Pamiętaj, że `<div>` to ostateczna wersja, gdy nic innego nie pasuje ;)